packages/bot/src/scratch/utils/scratch-helper.js

```diff
@@ -0,0 +1,22 @@
+import config                   from '../../constants';
+
+export const hasAllRequiredBlocks = () => {
```

Comment:
Can we pass in the workspace rather than using the global variable? This will keep the function pure. 

```suggestion
export const hasAllRequiredBlocks = (workspace) => {
```

Reply:
@aaron-binary this function is a scratch helper for current workspace which is only one that we already have it in a global variable. no need to pass it again.